lib/image-diff.js

```diff
@@ -81,8 +81,8 @@ ImageDiff.createDiff = function (options, cb) {
     }
 
     // Callback with pass/fail
-    var totalDifference = resultInfo[1];
-    return cb(null, totalDifference === '0');
+    var totalDifference = resultInfo[2]; // this is the percentage difference
+    return cb(null, parseFloat(totalDifference) <= options.threshold, options.diffPath);
```

Comment:
Oh, and we need to specify the radix for `parseFloat` to avoid parsing it as an octal.

```js
parseFloat(totalDifference, 10);
```